src/entities/users.entity.ts

```diff
@@ -34,6 +36,9 @@ class UserEntity extends BaseEntity {
 
   @UpdateDateColumn()
   updatedAt: Date;
+
```

Comment:
```
  @OneToMany(() => ConnectionsEntity, (connection) => connection.senderId)
  senderId: CommentEntity[];
```
```
  @OneToMany(() => ConnectionsEntity, (connection) => connection.receiverId)
  .receiverId: CommentEntity[];
```

Reply:
I think if we're going to create the connections entity, we should follow the bi-directional pattern documented [here](https://typeorm.io/many-to-many-relations).
Also you're referencing the wrong entity. (connections: CommentEntity[]) ?